cmd/tempo/app/modules.go

```diff
@@ -45,6 +45,7 @@ const (
 	Store         string = "store"
 	MemberlistKV  string = "memberlist-kv"
 	All           string = "all"
+	HA            string = "ha"
```

Comment:
ha-single-binary?
scaleable-single-binary?

just `ha` is not very descriptive.

Reply:
What do you think about just "scaleable"?

Reply:
I would like to communicate that it's a "single binary" or "all in one" kind of deployment.